lib/kernel/cuda/get_local_id.c

```diff
@@ -0,0 +1,38 @@
+/* OpenCL built-in library: get_local_id() for CUDA
```

Comment:
Isn't these identical to the "default implementations" in lib/kernel? In that case you can just use the generic source instead of copying it for CUDA. This has a CMake mechanism. See the other device libs.